run-tests.sh

```diff
@@ -1,53 +1,56 @@
 #!/usr/bin/env bash
 
-# NOTE: set TEST_SUITE to be markers you want to run.
+# Tested on debian and alpine
+# You will need to install some dependencies yourself.
 
 set -eo pipefail
 
 export PYTHONIOENCODING="utf-8"
 export LANG=C.UTF-8
-export PIP_PROCESS_DEPENDENCY_LINKS="1"
+
 # Let's use a temporary cache directory
 export PIPENV_CACHE_DIR=`mktemp -d 2>/dev/null || mktemp -d -t 'pipenv_cache'`
 
-prefix() {
-	sed "s/^/   $1:    /"
-}
+# on some Linux OS python is python3
+PYTHON=${PYTHON:-"python"}
+PIPENV_PYTHON="${PIPENV_PYTHON:-3.7}"
```

Comment:
Not sure I understand the value `PIPENV_PYTHON` here, because it defaults only to `-3.7` but it appears to be used in commands below.  Also, are we only running tests against python 3.7?

Reply:
The tests are running against all Python versions with every commit. This is done in:
https://github.com/pypa/pipenv/blob/638c84628df503e685c3ca80a9e72be95af13048/.github/workflows/ci.yaml#L42

The script is intended to be used by users locally. When you run the script you can override every variable with:
```
PIPENV_PYTHON=3.9 ./run-tests.sh
... results in:
Successfully installed wheel-0.37.1
$ PIPENV_PYTHON=3.9 python -m pipenv --venv && PIPENV_PYTHON=3.9 pipenv --rm
No virtualenv has been created for this project(/home/oznt/Software/pipenv) yet!
Aborted!
Installing dependencies...
$ PIPENV_PYTHON=3.9 python -m pipenv install --deploy --dev
```

Reply:
I think as long as python3.7 is supported the script should default to it. Also the CI pipeline is testing against this version.

Reply:
@matteius I added another commit with explicit explanation of the advanced usage of `run-tests.sh`